build.proj

```diff
@@ -18,8 +18,6 @@
     <RepoRoot>$(MSBuildThisFileDirectory)</RepoRoot>
     <CLIBuildFileName>$(RepoRoot)/build_projects/dotnet-cli-build/bin/dotnet-cli-build</CLIBuildFileName>
     <CLIBuildDll>$(CLIBuildFileName).dll</CLIBuildDll>
-    <BuildToolsDir>$(RepoRoot)/build_tools/</BuildToolsDir>
-    <ToolsDir>$(BuildToolsDir)/</ToolsDir>
 
```

Comment:
Because the properties aren't being used.  No reason to leave "junk" in the file.